### ISSUE TITLE
[BUG] Magma does not perform TwitchSpawn events

### ISSUE THREAD

**Issue by NovaPixell** [magmafoundation/Magma#292]
**Describe the bug**
I have encountered problems with the mod "TwitchSpawn", which can run events on Minecraft depending on what happens in the Twitch live.

**To Reproduce**
Install the mod on your Mohist Server.
Customize the config with your Minecraft nickname. Mod's Wiki: https://igoodie.gitbook.io/twitchspawn/twitchspawn-language/tsl-actions
/twitchspawn simulate {event:"donation", amount:1, currency:"EUR"}

**Expected behavior**
Basically the mod is loaded and read correctly, but Magma does not run these events which are triggered with events in the livestream. I ask if you would be able to fix this from the Magma side, as it works on a normal _forge server_.

**Info:**
Mod: twitchspawn_1.12.2-0.4.17

Debug Log when executing simulator command (with a custom rules file): https://pastebin.com/E3gTKXcQ

**Comment by kwright02** [comment]
```Player erenblaze is not found. Skipping ExecuteAction event.```

**Comment by kwright02** [comment]
This is a result of a misconfiguration on your part. Mod works fine for me.
